src/category_theory/arrow.lean

```diff
@@ -147,4 +147,31 @@ def map_arrow (F : C ⥤ D) : arrow C ⥤ arrow D :=
 
 end functor
 
+namespace helper
```

Comment:
No need for the namespace here. Probably should just be in the `arrow` namespace.

Reply:
Done.